functionary/functionary/settings/debug.py

```diff
@@ -4,10 +4,13 @@
 
 DEBUG = True
 
-INSTALLED_APPS += ["debug_toolbar"]
+INSTALLED_APPS += ["debug_toolbar", "django_browser_reload"]
```

Comment:
There's an issue either with this `django_browser_reload` app and middleware or our usage of it.  Basically every time you load a page in the UI (either by navigating or because of the new autoreload) a new database connection is opened.  The result is that just using functionary normally eventually exhausts the available connections to the database and the whole thing stops working.

We either need to figure out what's going on there and how to fix it, or remove this app.

Reply:
What did you use to get the connection count?

Reply:
> What did you use to get the connection count?

```sql
select count(*) from pg_stat_activity;
```

If you run that from a psql shell you'll notice that it goes up by 1 on every page load.  If you take `django_browser_reload` out of the picture that does not happen.

If this was an inherent problem with the library it would be unusable so I have to imagine it's some weird interaction between it and something else we're doing.  I haven't spent much time debugging it beyond noting described behavior, but we cannot have it enabled by default for our dev setup without figuring this out.

If we can't figure it out, then we can do the work to make it opt-in in the dev-setup scenario so that if some devs want to put up with the drawbacks they can, but not everyone is forced to.

Reply:
I'm not seeing this with either the middleware or the script tag in the home.html. I do briefly see an extra connection when making .py changes but the connection goes away in a reasonably short amount of time. Making changes to the .html shows up automatically as expected without extra connections.